### DISCUSSION TITLE
posting my work on github

### DISCUSSION THREAD

**Discussion by EOKELLO** [community/community#22276] [New to GitHub]
<p>how do i post my project on github</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/eokello">@eokello</a>,</p>
<p>Thanks for being here! To put your project up on GitHub, you’ll need to create a repository for it to live in.</p>
<p>You can get started with the following links:</p>
<ul>
<li><a href="https://help.github.com/en/articles/create-a-repo" rel="nofollow noopener">Create a repo</a></li>
<li><a href="https://help.github.com/en/articles/adding-an-existing-project-to-github-using-the-command-line" rel="nofollow noopener">Adding an existing project to GitHub using the command line</a></li>
</ul>
<p>I hope this helps!</p>
